vql/linux/connections.go

```diff
@@ -18,35 +18,76 @@
 package linux
 
 import (
+	"fmt"
 	"github.com/Velocidex/ordereddict"
 	"github.com/shirou/gopsutil/net"
 	"www.velocidex.com/golang/velociraptor/acls"
 	vql_subsystem "www.velocidex.com/golang/velociraptor/vql"
 	"www.velocidex.com/golang/vfilter"
 )
 
+var (
+	CommonName = map[string]string{
+		"11":  "UNIX_SOCK|SOCK_STREAM",
```

Comment:
It might be better to do this in VQL?

Then we can extend it when needed and it can work on old clients too.

Reply:
Good point, will close this PR and submit a new one on Monday.